jaxrs-clients/versions.lock

```diff
@@ -69,6 +69,12 @@
                 "com.palantir.remoting3:tracing"
             ]
         },
+        "com.github.ben-manes.caffeine:caffeine": {
```

Comment:
Think I'd prefer to keep the dependencies as minimal as possible. Guava's cache is probably good enough for this use case.